templates/statistics/statistics.html.twig

```diff
@@ -4,8 +4,43 @@
 
 {% block content %}
 
-    <h1 class="page-title">{% trans %}Statistics{% endtrans %}</h1>
-
     {% include '/includes/flash.html.twig' %}
 
+    <h1 class="page-title">{% trans %}Statistics{% endtrans %}</h1>
+    <table class="table"
+           data-toggle="table"
```

Comment:
@stejes die `data-` attributen mogen weg, is nodig voor de interactive tabel, maar dus niet voor hier.

Eventueel ook de header tekst centreren?
Eventueel ook een bordered table nemen https://getbootstrap.com/docs/4.0/content/tables/#bordered-table ?